### ISSUE TITLE
Basketball Hoop out of place 

### ISSUE THREAD

**Issue by TheRespectedMan** [OracleStation/OracleStation#522]
>https://i.imgur.com/bNEYSbl.png
It functions normally, but the sprite is off by 1 tile. Pic taken from metastation. 
Feels weird scoring goals on plating instead of on the hoop itself, y'know?

**Comment by SamHPurp** [comment]
I don't think I understand. Thats it gone through the hoop and chillin on the floor - its jsut goofy perspective, no?

**Comment by TheRespectedMan** [comment]
Yeah the sprite is in the wrong place, the wrong tile, but the hoop itself is in their intended tile (the plating).
